src/modules/main/store/actions.ts

```diff
@@ -1,15 +1,15 @@
+import { IPost } from '../../../lib/types';
 import {
   LoadData,
   MainActions,
   IExpertsItem,
-  IImportantItem,
   INewestItem,
 } from './actionTypes';
 
-export function loadImportant(payload: IImportantItem[]): MainActions {
+export function loadImportant(payload: IPost[]): MainActions {
   return {
     type: LoadData.LOAD_IMPORTANT,
-    value: payload,
+    payload,
   };
 }
```

Comment:
`payload` 
should be obtainer inside the `loadImportant` function - this is the place where you have to load the data